internal/state.go

```diff
@@ -30,17 +30,10 @@ type State struct {
 }
 
 func logConnectError(err error, engine string) {
-
-	if engine == "" {
-		fmt.Println("Could not connect to the default engine on http://localhost:9076")
-		fmt.Println("Specify where the engine is running using the --engine parameter or in your config file.")
-		fmt.Println("Error details: ", err)
-	} else {
-		fmt.Println("Could not connect to engine on " + engine + ".")
-		fmt.Println("Please check the --engine parameter or your config file.")
-		fmt.Println("Error details: ", err)
-	}
-	os.Exit(1)
+	msg := fmt.Sprintf("could not connect to engine on %s\nDetails: %s\n", engine, err)
+	msg += fmt.Sprintln("Please check if the engine is running and that the url specified")
+	msg += fmt.Sprintln("using either the --engine flag or the config file is correct.")
```

Comment:
Last sentence is a bit tricky, at least for Monday morning

Reply:
This could be because:
  the engine isn't up and running                                                                                                                              
  incorrect url in --engine flag or 'engine' property (in config)